closed/src/java.base/share/classes/sun/security/ec/NativeECUtil.java

```diff
@@ -217,4 +218,17 @@ static long getPrivateKeyNativePtr(ECPrivateKey key) {
             return nativePointer;
         }
     }
+
+    public static boolean isBrainpoolP512r1(ECParameterSpec name) {
+        NamedCurve curve = CurveDB.lookup(name);
+        if (curve != null) {
+            String[] nameAndAliases = curve.getNameAndAliases();
+            for (String nameAndAlias : nameAndAliases) {
```

Comment:
The local refers to a name _**or**_ an alias; suggest `nameOrAlias` is a better name.

Reply:
Updated to use `nameOrAlias`